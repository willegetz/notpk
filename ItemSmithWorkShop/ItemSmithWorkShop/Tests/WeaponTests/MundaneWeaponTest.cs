using ItemSmithWorkShop.AdventureItems.MundaneWeaponUtilites;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using ApprovalTests;
using ItemSmithWorkShop.WeaponUtilities;

namespace ItemSmithWorkShop.Tests.WeaponTests
{
	[TestClass]
	public class MundaneWeaponTest
	{
		[TestMethod]
		public void TestCreateWeapon()
		{
			var weapon = WeaponItemSmith.OrderItem("Dagger");
			Approvals.Approve(DisplayUtilities.BasicDisplay(weapon));
		}
	}
}
