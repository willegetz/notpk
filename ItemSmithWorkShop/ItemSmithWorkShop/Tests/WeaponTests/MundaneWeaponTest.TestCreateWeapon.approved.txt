Dagger:	'2 gp'
Weight: '1 pound(s)'
To Hit: ''
Damage: '1d4'
Hardness: '10'
Hit Points: '2'
	The dagger is a common secondary weapon. You get a +2 bonus on
	Sleight of Hand checks made to conceal a dagger on your body.